### DISCUSSION TITLE
Access token question

### DISCUSSION THREAD

**Discussion by rogers14k** [microsoftgraph/msgraph-sdk-powershell#2162] [Q&A]
Does running Revoke-MgUserSignInSession also revoke the access token? In the audit logs and from documentation it appears to only revoke the refresh token.

https://learn.microsoft.com/en-us/powershell/module/microsoft.graph.users.actions/revoke-mgusersigninsession?view=graph-powershell-1.0

**Comment by RayGHeld** [comment]
An access token is a static string that can be stored anywhere and would require the issue to keep track of them.  There isn't a way to revoke access tokens that I am aware of.
